core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -861,6 +873,10 @@ object KafkaConfig {
       .define(TransactionsAbortTimedOutTransactionCleanupIntervalMsProp, INT, Defaults.TransactionsAbortTimedOutTransactionsCleanupIntervalMS, atLeast(1), LOW, TransactionsAbortTimedOutTransactionsIntervalMsDoc)
       .define(TransactionsRemoveExpiredTransactionalIdCleanupIntervalMsProp, INT, Defaults.TransactionsRemoveExpiredTransactionsCleanupIntervalMS, atLeast(1), LOW, TransactionsRemoveExpiredTransactionsIntervalMsDoc)
 
+    /** ********* Fetch Session Configuration **************/
+      .define(MaxIncrementalFetchSessionCacheSlots, INT, Defaults.MaxIncrementalFetchSessionCacheSlots, atLeast(0), MEDIUM, MaxIncrementalFetchSessionCacheSlotsDoc)
+      .define(MinIncrementalFetchSessionEvictionMs, LONG, Defaults.MinIncrementalFetchSessionEvictionMs, atLeast(0), MEDIUM, MinIncrementalFetchSessionEvictionMsDoc)
```

Comment:
I thought we agreed in the KIP that this will be a constant and not configurable?

Reply:
Good catch, will make this a constant.